dev/release_files/LICENSE

```diff
@@ -201,28 +201,1238 @@
+Copyright (c) 2013 Nicolas Bouillon
+
+angular-ui-bootstrap (http://angular-ui.github.io/bootstrap/) Copyright (c) 2012-2016 the AngularUI
+Team, https://github.com/organizations/angular-ui/teams/291112
+
+Font Awesome CSS (http://fontawesome.io/).
+
+JRuby jcodings 1.0.8 (https://github.com/jruby/jcodings)
+
+JRuby joni 2.1.2 (https://github.com/jruby/joni)
+
+SLF4j (http://slf4j.org) Copyright (c) 2004-2013 QOS.ch
+
+HSQLDB (http://hsqldb.org/)
+
+  For work developed by the HSQL Development Group:
+  Copyright (c) 2001-2016, The HSQL Development Group
+
+  For work originally developed by the Hypersonic SQL Group:
+  Copyright (c) 1995-2000 by the Hypersonic SQL Group.
+
+---
+
+This product bundles Font Awesome fonts (http://fontawesome.io/) which is licensed
+under the SIL OFL 1.1 License.
+
+---
+
+This product contains the following products which are licensed with the Sun Public
+License (SPL).
+
+BeanShell (http://www.beanshell.org)
+
+---
+
+This product contains the following products which are in the Public Domain.
+
+Aopalliance 1.0 (http://aopalliance.sourceforge.net/)
+
+---
+
+This product contains the following products which are licensed under the Common
+Development and Distribution License (CDDL).
+
+Jersey 1.9 (https://jersey.java.net/) Copyright (c) 2010-2011 Oracle and/or its affiliates.
+
+Jaxb 2.2.3-1 (https://jaxb.java.net/) Copyright (c) 2010 Oracle and/or its affiliates.
+
+JavaBeans Activation Framework 1.1 (http://java.sun.com/products/javabeans/jaf/index.jsp) Copyright 1997-2005 Sun Microsystems, Inc.
+
+JavaServer Pages(TM) API 2.1  (https://jsp.java.net/) Copyright ©2013 Oracle Corporation.
+
+Java(TM) Servlet API Design Specification 3.0.1 (http://servlet-spec.java.net/) Copyright (c) 1997-2011 Oracle and/or its affiliates.
+
+JavaServlet(TM) Specification 2.5 (https://jcp.org/aboutJava/communityprocess/mrel/jsr154/index.html) Copyright 2005 Sun Microsystems, Inc.
+
+JAXB API bundle for GlassFish V3 2.2.2 (https://jaxb.dev.java.net/) Copyright (c) 2010 Oracle and/or its affiliates
+
+Streaming API for XML 1.0-2 Copyright (c) 2003 by BEA Systems
+
+---
+
+This product contains the following products which are licensed under the
+Apache Software License, version 1.1.
+
+Apache Commons EL 1.0 (https://commons.apache.org/proper/commons-el/)
+
+Apache Jakarta Oro 2.0.8 (http://jakarta.apache.org/)
+
+---
+
+This product bundles the following products which are licensed under the
+Eclipse Public License (EPL):
+
+JUnit 4.12 (http://junit.org/junit4/) Copyright © 2002-2016 JUnit.
+
+---
+
+This product includes the following products which are licensed using the
+Creative Commons Public License:
+
+Apache HttpComponents
+
+  This project contains annotations in the package org.apache.http.annotation
+  which are derived from JCIP-ANNOTATIONS
+  Copyright (c) 2005 Brian Goetz and Tim Peierls.
+  See http://www.jcip.net and the Creative Commons Attribution License
```

Comment:
Above we say "Creative Commons Public License" and here "Creative Commons Attribution", which is it?